robot/basestation/templates/pages/Rover.html

```diff
@@ -1,28 +1,30 @@
 {% extends "pages/Base.html" %}
 {% block title %}Rover{% endblock %}
 {% block head %}
- {{ super() }}
-   <script src="../../static/js/pages/rover.js"></script>
-   <script src="../../static/js/elements/feed/console.js"></script>
-   <script src="../../static/js/elements/navigation/gps.js"></script>
-   <script src="../../static/js/elements/navigation/navigation.js"></script>
-   <script src="../../static/js/elements/controls/camera-controls.js"></script>
-   <script src="../../static/js/elements/feed/camera.js"></script>
-   <script src="../../static/js/elements/controls/extra-controls.js"></script>
+{{ super() }}
+    <script src="../../static/js/pages/rover.js"></script>
+    <script src="../../static/js/elements/feed/console.js"></script>
+    <script src="../../static/js/elements/navigation/navigation.js"></script>
+    <script src="../../static/js/elements/controls/camera-controls.js"></script>
+    <script src="../../static/js/elements/feed/camera.js"></script>
+    <script src="../../static/js/elements/controls/extra-controls.js"></script>
+    <script src="../../static/js/p5.min.js"></script>
+    <script src='../../../static/js/elements/navigation/localization.js'></script>
+    <script src='../../../static/js/elements/navigation/nav_funs.js'></script>
```

Comment:
1. The semantics of `nav_funs.js` can be improved.
2. We are currently at war with inconsistent naming practices of files/folders, so we have setup some [guidelines](https://github.com/space-concordia-robotics/robotics-prototype/wiki/Code-Formatting-and-Conventions#filedirectory-names) to combat this.

I suggest renaming the file `NavigationUtilities.js`, or something of the sort.